src/components/Sing/ScoreSequencer.vue

```diff
@@ -475,6 +475,10 @@ const previewMove = () => {
     }
   }
   for (const note of editedNotes.values()) {
+    if (note.noteNumber < 0 || note.noteNumber >= keyInfos.length) {
+      return;
```

Comment:
下にコメントがあったのでこちらも真似てみます 🙏 
```suggestion
      // MIDIキー範囲外へはドラッグしない
      return;
```